Jenkinsfile

```diff
@@ -2,11 +2,12 @@ pipeline {
   environment {
     GET_CONTENT = "true"
     NODE_ENV = "production"
+    HOME = "${env.WORKSPACE}"
     TZ = "UTC"
   }
 
   agent {
-    label 'docker&&linux'
+    dockerfile true
```

Comment:
@MarkEWaite whats the recommended way to do this without using the s390? scripted?

```
12:04:30  npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
12:04:30  npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.11.3/libvips-8.11.3-linux-s390x.tar.br
12:04:30  npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
12:04:30  npm ERR! sharp: Installation error: Prebuilt libvips 8.11.3 binaries are not yet available for linux-s390x
```

https://ci.jenkins.io/job/Infra/job/plugin-site/job/PR-898/2/console